packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics.util.ts

```diff
@@ -198,9 +198,16 @@ export const isBrowserMediaErrorName = (errorName: any) => {
  * @returns
  */
 export const getBuildType = (
+  webex,
   webClientDomain,
   markAsTestEvent = false
 ): Event['origin']['buildType'] => {
+  webex.logger.log(
```

Comment:
`getBuildType()` is called lots of times, we don't want to be logging every time it's called as this will flood the log with unnecessary log lines.
Wouldn't it be better to just move the `getBuildType()` function into a separate file like util.ts or something so than there is no problem with spying or mocking it in unit tests.

Reply:
@marcin-bazyl - If you notice, it is already in a util file. If we move it to another util file, it might not look good on the code. That is why I chose to do it this way. 

Is it okay to have it in another utils file?

Reply:
Ah. That makes sense too. I'll try to do that @marcin-bazyl

Reply:
Done, @marcin-bazyl